pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -88,12 +102,18 @@ stdenv.mkDerivation rec {
       --replace "${abseil.url}" "${abseil}"
   '';
 
+  postBuild = lib.optionalString pythonSupport ''
+    ${python3Packages.python.interpreter} ../setup.py bdist_wheel
+  '';
+
   postInstall = ''
     # perform parts of `tools/ci_build/github/linux/copy_strip_binary.sh`
     install -m644 -Dt $out/include \
       ../include/onnxruntime/core/framework/provider_options.h \
       ../include/onnxruntime/core/providers/cpu/cpu_provider_factory.h \
       ../include/onnxruntime/core/session/onnxruntime_*.h
+  '' + lib.optionalString pythonSupport ''
+    pip install dist/*.whl --no-index --no-warn-script-location --prefix="$python" --no-cache --no-deps
```

Comment:
@jonringer @mweinelt we might be able to use the pipInstallHook here if we make the prefix variable as well as the phase (this is related again to sorted phases).

Reply:
configurable?

Reply:
sounds good to me, if you can get it to work.